packages/website/.gitignore

```diff
@@ -1,2 +1,5 @@
 public/sitemap.xml
-public/robots.txt
\ No newline at end of file
+public/robots.txt
```

Comment:
@orvn what happened to the robots 🤖 ? Do we still have one?

Reply:
@orvn plz give background on robots.txt removal. Is deliberate or lost in the refactor?